src/mumble/Settings.cpp

```diff
@@ -644,6 +645,7 @@ void Settings::load(QSettings* settings_ptr) {
 	SAVELOAD(iMaxImageWidth, "net/maximagewidth");
 	SAVELOAD(iMaxImageHeight, "net/maximageheight");
 	SAVELOAD(qsServicePrefix, "net/serviceprefix");
+	SAVELOAD(iMaxInFlightTCPPings, "net/maxinflighttcppings");
```

Comment:
Given the error message we use (`Server is not responding to TCP pings`), maybe we should check the setting we load is > 0? Or do we expect user who hack registry to know what they are doing? Would be fine by me.